action.yml

```diff
@@ -6,18 +6,27 @@ inputs:
     description: 'Which version of the Dune binary to install'
     required: false
     default: 'nightly'
-  automagic:
-    description: 'Whether to run automatically all the standard lock, build, test steps'
-    required: false
-    default: 'false'
   steps:
     description: 'Fine-grain control of the steps to run; override `automagic` when set'
     required: false
-    default: ''
+    default: 'all'
   directory:
     description: 'Directory in which dune should be run'
     required: false
     default: '.'
+  workspace:
+    description: 'Path of a `dune-workspace` file (in `directory`)'
+    required: false
+    default: ''
+  display:
+    description: 'Control the log format generated by dune, see dune-config(5)'
+    required: false
+    default: ''
+  cache:
+    description: 'Enable the cache of dune’s cache'
```

Comment:
Maybe "Store and retrieve the Dune cache in a cache to enable caching between action runs" or something like that?